tests/tests_integration/test_api/test_simulators/test_models.py

```diff
@@ -56,7 +56,7 @@ def test_list_model_revisions(self, cognite_client: CogniteClient, seed_resource
         )
 
         model_revision_ids = []
-        for revision in cognite_client.simulators.models.revisions(limit=2):
+        for revision in cognite_client.simulators.models.revisions.list(limit=2):
```

Comment:
```suggestion
        for revision in cognite_client.simulators.models.revisions(limit=2):
```
this is exactly the method that we want to test, why changed?